Sources/MuxUploadSDK/InternalUtilities/UploadEvent.swift

```diff
@@ -43,7 +43,7 @@ struct UploadEvent: Codable {
 
         self.deviceModel = device.model
 
-        self.appName = Bundle.main.appName
```

Comment:
Does this result in a different value for `appName` for the same app?

Reply:
It does, but this is known. We want the unique name because it's easier to identify

Reply:
ok, just something to note if we're ever trying to correlate upload stats across versions